ocs_ci/helpers/helpers.py

```diff
@@ -3480,3 +3480,21 @@ def wait_for_rook_ceph_pod_status(pod_obj, desired_status, timeout=420):
             return False
 
     return True
+
+
+def check_mon_pods_eq_3():
+    """
+    Get number of monitoring pods
```

Comment:
Can explain bit more about the function.

Reply:
Done